src/utils/text.ts

```diff
@@ -100,3 +115,17 @@ export function normalizeInputString(str: string, wrapEnter = true): string {
     }
     return finalStr;
 }
+
+export function diffLineText(oldLine: string, newLine: string): TextChange {
```

Comment:
Ah really? I just rewrote it, I had no idea it was ever here :sweat_smile: 

I put it in `utils` mostly so it could be unit-tested. We can't run unit tests against `command_line_manager` (due to the `vscode` import), and making a separate module just for this one function felt a bit silly.

Reply:
Perhaps no need for unit tests? 🙈